mds_asssignment_1/guestbook/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils import timezone
+
+class Entry(models.Model):
+    fname = models.CharField(max_length=50, verbose_name='First Name')
+    lname = models.CharField(max_length=50, verbose_name='Last Name', blank=True)
+    email = models.CharField(max_length=100, blank=True)
+    comment = models.CharField(max_length=500)
```

Comment:
For these longer text fields it's normally better to use `models.TextField` which will use a `<textarea> in the admin and can optimise database storage:
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.TextField